pages/_app.js

```diff
@@ -8,8 +8,26 @@ export default function App({ Component, pageProps }) {
   const [donations, setDonations] = useLocalStorageState("donations", {
     defaultValue: [...userDonations],
```

Comment:
you don't need to spread the userDonations here in a new array, userDonations is already an array

Reply:
oh yes you are right, I can pass it directly..